backend/infrahub/message_bus/operations/check/artifact.py

```diff
@@ -24,11 +24,17 @@ async def create(message: messages.CheckArtifactCreate, service: InfrahubService
 
     if InfrahubKind.READONLYREPOSITORY:
         repo = await InfrahubReadOnlyRepository.init(
-            id=message.repository_id, name=message.repository_name, client=service.client
+            id=message.repository_id,
+            name=message.repository_name,
+            client=service.client,
+            service=service,
```

Comment:
Not strictly related to this PR but now that we start to require the service object for repositories I think we should also remove `client` as a parameter and transition to using the client provided by the service all of the time instead.